src/main/java/seedu/address/logic/commands/student/ExtractEmailsCommand.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.logic.commands.student;
+
+import java.util.stream.Collectors;
+
+import javafx.collections.ObservableList;
+import javafx.scene.input.Clipboard;
+import javafx.scene.input.ClipboardContent;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.model.Model;
+import seedu.address.model.student.Student;
+
+
+/**
+ * Shows student's grade distribution in ModQuik.
+ */
+public class ExtractEmailsCommand extends Command {
+
+    public static final String COMMAND_WORD = "extract emails";
+
+    public static final String MESSAGE_SUCCESS = "Copied link to clipboard! Open it in a browser.";
```

Comment:
Maybe you can add more guidance, for example: paste it in your browser and press Enter or Use CTrl+V ... since the user might not know what's a clipboard.